wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/romireference/commands/TurnTime.java

```diff
@@ -0,0 +1,63 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+/*
+ * Creates a new TurnTime command. This command will turn your robot for a 
+ * desired rotational speed and time.
+ */
+public class TurnTime extends CommandBase {
+    private final double m_duration;
+    private final double m_rotational_speed;
+    private final Drivetrain m_drive;
+    private long m_startTime;
+
+   /**
+   * Creates a new TurnTime.
+   *
+   * @param speed   The speed which the robot will turn. Negative is in reverse.
+   * @param duration_in_seconds   How long to turn for.
+   * @param drive   The drive subsystem on which this command will run
+   */
+    public TurnTime(double speed, double duration_in_seconds, Drivetrain drive) {
+        m_rotational_speed = speed;
+        m_duration = duration_in_seconds * 1000;
+        m_drive = drive;
```

Comment:
We need to `require(drive)` for the scheduler to handle passing control properly.

Reply:
Added.